Source/WebKitLegacy/mac/WebView/WebHTMLView.mm

```diff
@@ -6255,6 +6255,16 @@ - (WebPluginController *)_pluginController
     return _private->pluginController.get();
 }
 
+- (WebCore::ScrollbarWidth)_scrollbarWidthStyle
+{
+    auto* frame = core([self _frame]);
+
+    if (!frame || !frame->document() || !frame->document()->documentElement() || !frame->document()->documentElement()->renderer())
+        return WebCore::ScrollbarWidth::Auto;
+
+    return frame->document()->documentElement()->renderer()->style().scrollbarWidth();
```

Comment:
uses documentElement not scrollingElement as scrollingElement was wrong. Existing code fixed in https://github.com/WebKit/WebKit/pull/14429